### ISSUE TITLE
improve readme

### ISSUE THREAD

**Issue by emanuil-tolev** [g4he/g4he#192]
Hi @richard-jones @markmacgillivray 

If one has:
a/ all the data in ES
b/ the source code from this repo

but the collaboration reports are showing "no data available" - are there any scripts to run or anything else to do?

Newest deployment is at http://g4he.cottagelabs.com/organisation/Brunel%20University and says 
"Brunel University has 244 projects, worth a total of £113,014,351, involving 407 collaborators"

but the reports show nothing.

New potential discovery **WORKS**. For Brunel I can go type in "urban development" and it will make a relevant report just like before.

**Comment by emanuil-tolev** [comment]
If you want to see what's in ES: http://178.62.234.142:9200

**Comment by richard-jones** [comment]
Nope, should all work.  Just looking at the network traffic, the requests
to the various endpoints are coming back empty, so looks to be a problem on
the server end.

On 21 September 2014 12:37, Emanuil Tolev notifications@github.com wrote:

> If you want to see what's in ES: http://178.62.234.142:9200
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/g4he/g4he/issues/192#issuecomment-56296246.

## 

Richard Jones,

Founder, Cottage Labs
t: @richard_d_jones, @cottagelabs
w: http://cottagelabs.com

**Comment by emanuil-tolev** [comment]
Currently being resized so it will be down for a bit. But yes, the points return empty lists instead of results.

But why, I copied the elasticsearch (literally, the executables with the data folder) as well as the code repo. I didn't even just pull it, I copied the exact repo from the old server to the new one. I'm running master anyway, and have all the submodules updated (I decided to get the latest code from here in the end).

I'll have to check what the code does and why it's coming back empty.

**Comment by emanuil-tolev** [comment]
Resolved - add to README "Requires dynamic scripting in Elasticsearch"

@markmacgillivray @richard-jones can I get push access to this repo? I have a few minor changes in a fork, I'll make a pull req and merge them when I get access.
